code/controllers/subsystem/blackbox.dm

```diff
@@ -267,28 +275,51 @@ Versioning
 		first_death["area"] = "[AREACOORD(L)]"
 		first_death["damage"] = "<font color='#FF5555'>[L.getBruteLoss()]</font>/<font color='orange'>[L.getFireLoss()]</font>/<font color='lightgreen'>[L.getToxLoss()]</font>/<font color='lightblue'>[L.getOxyLoss()]</font>/<font color='pink'>[L.getCloneLoss()]</font>"
 		first_death["last_words"] = L.last_words
+	var/area/placeofdeath = get_area(L)

```

Comment:
this is only use for the name an area. might as well switch it to get_area_name(L, TRUE)